README.md

```diff
@@ -454,10 +313,40 @@ rather than having to ignore every instance of that method:
 }
 ```
 
+## Accurate stack traces using source-maps
```

Comment:
I realize you didn't write this section / it's just been moved but from what I can tell it does not work at all.  The problem is that by default source-map-support does not support inline source-maps (like those produced by nyc itself).  Interesting note `@babel/register` runs `source-map-support` in a way which finds the source-map's produced by babel, so AFAIK right now the only way to get accurate stack traces is to have nyc use `@babel/register` + `babel-plugin-istanbul`.

See https://github.com/evanw/node-source-map-support/issues/239

Reply:
Should I just delete it for now? I was on the fence about it since I wasn't sure, and I think it warrants a more advanced doc later on when that issue is resolved

Reply:
I'm leaning towards yes.

Reply:
I agree .. the more work I do on source-maps, the more I realize this is something we should try to patch in Node.js itself. This would at least, hopefully, mean folks using Node 12.x would eventually have better stack traces.